src/components/Verse/OverflowVerseActionsMenu.tsx

```diff
@@ -35,6 +35,13 @@ const OverflowVerseActionsMenu: React.FC<Props> = ({
   bookmarksRangeUrl,
 }) => {
   const { t } = useTranslation('common');
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const closePopover = () => {
+    setIsPopoverOpen(false);
+    return onActionTriggered;
```

Comment:
Salam alykum @afilahkle,

JAK for your amazing contribution, much appreciated! The PR looks amazing masha'Allah; I just have a couple of comments.

Shouldn't this be `onActionTriggered()` instead of `return onActionTriggered;`?

Reply:
walekum assalam @ahmedriad1,

JAK for your reply, i tried it but i'm getting this error

![Screenshot from 2023-05-20 20-22-55](https://github.com/quran/quran.com-frontend-next/assets/90985041/86860a38-d6bd-4545-b324-e0e8342d7862)

how to fix it, or should i return it.

Reply:
oh, I think you might need to wrap it with an `if` statement:
```ts
if (onActionTriggered) {
  onActionTriggered();
}
```

Reply:
great, will do that.